app/user/tests/test_views.py

```diff
@@ -148,3 +151,99 @@ def test_profile_update(self, upload_resource: Any) -> None:
 
         self.assertTrue(upload_resource.called)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+class TestPasswordReset(TestCase):
+    testuser: dict
+    user: Any
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.testuser = {
+            "username": fake.user_name(),
+            "email": fake.email(),
+            "password": fake.password(),
+        }
+        cls.user = User.objects.create_user(**cls.testuser)
+        return super().setUpClass()
+
+    def test_password_reset_request(self) -> None:
+        url = reverse("password-reset")
+        data = {"email": self.testuser["email"]}
+        outbox = len(mail.outbox)
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(mail.outbox), outbox + 1)
+
+    def test_password_reset_none_existing_email(self) -> None:
+        url = reverse("password-reset")
+        data = {"email": fake.email()}
+        outbox = len(mail.outbox)
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(mail.outbox), outbox + 0)
```

Comment:
why are you douing + 0?? remove the plus 0

Reply:
Okay